modules/installation-supported-aws-machine-types.adoc

```diff
@@ -13,7 +13,7 @@
 
 The following Amazon Web Services (AWS) instance types are supported with {product-title}.
 
-.Instance types for machines
+.Instance types for x86_64 machines
```

Comment:
One observation - most references I have found are to the x86_64 architecture. Might just want to check that `x86_64 machines` is a sound reference.

Reply:
+1 to @kmccarron-rh

Reply:
maybe "machines based on x86_64 architecture" ? also i'm not sure if the usual doc nomenclature is to refer to the arch by go's amd64 or x86_64 like the kernel.